mailpoet/lib/Form/DisplayFormInWPContent.php

```diff
@@ -160,6 +160,7 @@ private function getContentBellow(FormEntity $form, string $displayType): string
     $templateData['animation'] = $formSettings['form_placement'][$displayType]['animation'] ?? '';
     $templateData['fontFamily'] = $formSettings['font_family'] ?? '';
     $templateData['enableExitIntent'] = false;
+    $templateData['cookieFormExpirationTime'] = $formSettings['cookie_expiration_time'] ?? 365;
```

Comment:
Is the value of 365 only for backwards compatibility? It's not possible to set such a value from the UI.

Reply:
I switched 365 to 7 (new default) on a later commit and added a comment about the backwards compatibility.
Sorry for the confusion.